server/fleet/datastore.go

```diff
@@ -399,7 +399,7 @@ type Datastore interface {
 	// QueryResultsStore
 
 	// QueryResultRows returns all the stored results of a query (from all hosts).
```

Comment:
Tiny nit: should we update this comment to mention that it's all hosts that the user has access to?

Reply:
It's dependent on what's passed into TeamFilter here, but that's a good call i'll add that
comment into the interface for GetQueryReportResults